package.json

```diff
@@ -31,18 +31,24 @@
     "@babel/preset-env": "^7.14.5",
     "@babel/types": "7.12.*",
     "@types/argparse": "^1.0.10",
+    "@types/copy-webpack-plugin": "^8.0.0",
+    "@types/css-minimizer-webpack-plugin": "^3.0.1",
     "@types/lodash": "^4.14.170",
+    "@types/mini-css-extract-plugin": "^1.4.3",
     "@types/node": "^15.12.2",
     "@types/node-fetch": "^2.5.10",
     "@types/tar-fs": "^2.0.0",
+    "@types/webpack": "^5.28.0",
+    "@types/webpack-dev-server": "^3.11.4",
     "@typescript-eslint/eslint-plugin": "^4.15.0",
     "@typescript-eslint/parser": "^4.15.1",
     "@zoltu/typescript-transformer-append-js-extension": "^1.0.1",
     "babel-loader": "^8.2.2",
     "chai": "^4.2.0",
     "chalk": "^2.4.2",
-    "clean-webpack-plugin": "^3.0.0",
+    "clean-webpack-plugin": "^4.0.0-alpha.0",
```

Comment:
Did you need something from this alpha version?

Reply:
The exported class of v3.x is not compatible with the plugin interface of webpack 5, I didn't investigate why but update to 4.0.0 fixed this.